platform/persistence/platform-persistence-core-impl/pom.xml

```diff
@@ -202,7 +202,6 @@
                             javax.xml.xpath,
                             org.apache.commons.logging,
                             org.codice.ddf.persistence;version=1.0.0-SNAPSHOT,
-                            org.geotools.factory,
```

Comment:
Oh i should put this back. It was just a test to see if we *really* need to explicitly import (surprise, we don't) but i'm going to leave it all in here for now. Minimal changes and all. we can make a ddf ticket to refactor these poms if we want